modules/understanding-upgrade-channels.adoc

```diff
@@ -7,116 +7,99 @@
+[id="update-recommendation-removal"]
+== Update recommendation removals
+Red Hat monitors newly released versions and update paths associated with those versions both before and after they're added to supported channels. When a serious regression is identified Red Hat may remove affected update recommendations. If Red Hat chooses to remove update recommendations that action is taken in all relevant channels, simultaneously. The removal of recommended updates may happen either before or well after updates have been offered in supported channels.
+
+In the event that Red Hat removes update recommendations from any supported release we will eventually provide a superseding update paths to a future version which corrects the regression. There may however be a delay while the defect is corrected, tested, and promoted.
+
+When update recommendations are removed from supported channels those update paths remain supported but are no longer recommended. All supported but not recommended update paths updating from {product-title} version 4.10 and later will have a descriptive message and reference URL which links to more thorough explanation for the recommendation removal. Please review the linked reference and carefully consider whether you wish to continue with an update which is no longer recommended. See link:conditional-updates-overview_{context}[Conditional Updates] section for more information on this feature.
 
-Red Hat will eventually provide supported update paths from any supported release in the `fast-{product-version}` or `stable-{product-version}` channels to the latest release in {product-version}.z, although there can be delays while safe paths away from troubled releases are constructed and verified.
-endif::openshift-origin[]
 
-ifdef::openshift-origin[]
-The presence of an update recommendation in the `stable-4` channel at any point is a declaration that the update is supported. While releases will never be removed from the channel, update recommendations that exhibit serious issues will be removed or made conditional from all channels. When an update recommendation is supported, it remains supported for the life of {product-version}, even if the update recommendation is later dropped or made conditional.
-endif::openshift-origin[]
 
 ifndef::openshift-origin[]
 
 [id="fast-stable-channel-strategies_{context}"]
-== Fast and stable channel use and strategies
+== Choosing the correct channel for your cluster
+
+Because update recommendations available in both `fast-{product-version}` and `stable-{product-version}` are fully supported and benefit equally from extensive data analysis the choice between the two channels should be guided based on your specific needs rather than which channel is the default or qualities implied by the channel names. Largely this is a choice between updating to the latest GA versions or waiting while Red Hat collects additional data before promoting to the stable channel.
```

Comment:
"Because update recommendations available in both `fast-{product-version}` and `stable-{product-version}` are fully supported and benefit equally from the extensive testing and qualification Red Hat performs prior to release, the choice....."

We can't say they equally benefit from data analysis because there is a differing level of analysis carried out.

Reply:
Yeah it's a very nuanced point. Things make it to fast before we have data and we've analyzed it, however as soon as we decide a release is bad, whether it's made it to the stable channel or not, we remove update recommendations everywhere including the fast channel.

I really struggle with how to make this point clear but I think it's valuable to explain because I believe it bolsters the notion that fast is faster but not necessarily less stable. Other ways I've thought of are along the lines of "Fast channel contains no known serious regressions, the difference with the stable channel is that not only is it free of known serious regressions, but we have collected sufficient data that if there were detectable regressions we should know about them."

Reply:
We should try to set the tone throughout that `fast` is THE channel to use and that `stable` is an "add on" for those  exceptional cases where someone can wait. Unfortunately though, `stable` is the default which kind of runs counter to that strategy.